### DISCUSSION TITLE
mod "meta"

### DISCUSSION THREAD

**Discussion by TheExecutionist0** [eaglerforge/EaglerForge#14] [Ideas]
Would it be possible to add stuff like a mod's icon (image could be converted into base64), the developer or owner of the mod, mod description, ect.

**Comment by radmanplays** [comment]
next update

**Comment by stpv221** [comment]
I agree. this would be cool

**Comment by STPv22** [comment]
:( 
![image](https://github.com/user-attachments/assets/c4a60686-def8-4b6a-b245-6f0d62347e30)

**Reply by leahanderson1** [reply]
not really what i meant

**Reply by Project516** [reply]
wHaT aBoUt DuCkDuCkGo???

**Reply by radmanplays** [reply]
> wHaT aBoUt DuCkDuCkGo???
> 

that's not even a browser, it's a search engine

**Reply by STPv22** [reply]
> > wHaT aBoUt DuCkDuCkGo???
> 
> that's not even a browser, it's a search engine

bUt WHaT AbOuT eCoSiA

**Reply by STPv22** [reply]
> not really what i meant

ya i kind of realized that that wasn't how it worked

**Reply by Project516** [reply]
> > wHaT aBoUt DuCkDuCkGo???
> 
> that's not even a browser, it's a search engine

<img width="1180" alt="image" src="https://github.com/user-attachments/assets/000104ff-11b9-4f5d-aacf-ff86e8ad745d">
Umm it clearly is a browser
